### PR TITLE
Let's start building Issue 4!

### PR DESCRIPTION
**Pull request by gwijthoff**

### CODE REVIEW

**Review by rlskoeser — COMMENTED** [review]
Updates look good (based on reviewing on github, haven't pulled to run locally), can't believe you got all this added so fast.

What is the font change for?

Could you rename images to remove the `'` characters?

File: themes/startwords/assets/scss/article/_single.scss [hunk 1]
```diff
@@ -147,7 +147,7 @@ body.article article, body.page article {
             p {
                 margin: 0;
                 font-style: italic;
-                font-weight: 400;
+                font-weight: 300;
```
[the remark this answers is not in the document]
Comment (gwijthoff) [reply themes/startwords/assets/scss/article/_single.scss:150]: @rlskoeser Before this issue, we haven't had articles that use many block quotes. Seeing many of them here made me realize that for some reason, we have block quotes set to a heavier font weight (400) than the rest of the body text (300), in addition to being italicized. This made the block quotes jump out from the rest of the article. Typographic style guides generally say that block quotes should only be set off with one typographic difference: italics, or indentation, or smaller font, but never more than one change.

[the remark this answers is not in the document]
Comment (rlskoeser) [reply themes/startwords/assets/scss/article/_single.scss:150]: ah, ok. thanks for clarifying - I didn't expand to see the full context, that this was only for block quotes.

File: themes/startwords/assets/scss/article/_single.scss
Comment (gwijthoff) [inline themes/startwords/assets/scss/article/_single.scss:1]: Previously, we had block quotes set to a heavier font weight (400) than the rest of the body text (300), in addition to being italicized. This made the block quotes jump out from the rest of the article. Typographic style guides generally say that block quotes should only be set off with one typographic difference: italics, or indentation, or smaller font, but never more than one change.

**Pull request merged**